### ISSUE TITLE
3 "unexpected error has occured" dialogs with logs related to NvThermalSensors after every wake from sleep

### ISSUE THREAD

**Issue by knowsshit** [Rem0o/FanControl.Releases#2292]
**Describe the bug**
Every time after Windows wakes from sleep, FanControl pops up and I have to dismiss three identical error dialogs with "An unexpected 
![Screenshot 2024-02-04 114137](https://github.com/Rem0o/FanControl.Releases/assets/93446682/c59f08c5-3d0b-41ee-a5d7-b77b9ea8d756)
error has occured and has been added to the log file."

**Is there a log.txt file next to FanControl.exe with recent date entries?**
[log.txt](https://github.com/Rem0o/FanControl.Releases/files/14156450/log.txt)

**Relevant hardware specs and setup**
[Laptops and most proprietary pre-built dedsktops are not supported.]
Asus ROG Z790 HERO, i9-13900k, Asus ROG STRIX 4090, Windows 11, latest updates and drivers.

The GPU og GPU Hot Spot sensors seems to work fine, so I just want the errors supressed so I don't have to have FanControl popping up and making me dismiss the three error dialog boxes.

**Comment by Rem0o** [comment]
> System.NullReferenceException: Object reference not set to an instance of an object.
at FanControl.NvThermalSensors.NvThermalSensors.GetSensorValue(NvPhysicalGpuHandle handle, UInt32 mask, Int32 index)

Seems like a null reference in the plugin's code? The fix has to be made in there.
https://github.com/TimSirmovics/FanControl.NvThermalSensors

**Comment by Rem0o** [comment]
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/0b4cfd78-a98c-4dbf-adb7-65bad1a0443e)

I can easily see a null ref possibility right here.

**Comment by knowsshit** [comment]
Thanks for the quick reply! I have no idea how to fix that myself, but I opened an issue for FanControl.NvThermalSensors at https://github.com/TimSirmovics/FanControl.NvThermalSensors/issues/1

**Comment by knowsshit** [comment]
Would it be possible to have an option for FanControl to ignore such errors from plugins?

**Comment by knowsshit** [comment]
It is now fixed in new release of NVThermalSensors: https://github.com/TimSirmovics/FanControl.NvThermalSensors/releases

I do however remember that if I tried to click away the error messages while FanControl was minimized, they wouldn't go away. I had to open the FanControl window and then click on the error messages. Not sure if that is another bug or if it is fixed or not. Just thought I would mention that as well.
